doc/nrf/protocols/matter/index.rst

```diff
@@ -20,9 +20,9 @@ The |NCS| allows you to develop applications with different versions of Matter,
 +--------------------------+-----------------------------------------------------+------------------------+
 | nRF Connect SDK version  | Matter specification version                        | `Matter SDK version`_  |
 +==========================+=====================================================+========================+
-| v2.5.99 (latest)         | :ref:`1.2.0 <ug_matter_overview_dev_model_support>` | 1.2.0.1                |
+| |release|                | :ref:`1.2.0 <ug_matter_overview_dev_model_support>` | 1.2.0.1                |
 +--------------------------+-----------------------------------------------------+------------------------+
-| |release|                | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
+| v2.5.2                   | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
```

Comment:
@wiba-nordic Is it appropriate regarding the NCS 2.6.0 release?

Reply:
Yes, I neede to think about this for a bit myself. After the release, we'll re-add an entry for latest.

Reply:
I mean, this should replace itself with 2.6.0 automatically after the release tag, am I right?

Reply:
Correct.